tools/nni_cmd/nnictl_utils.py

```diff
@@ -736,3 +737,57 @@ def search_space_auto_gen(args):
         print_warning('Expected search space file \'{}\' generated, but not found.'.format(file_path))
     else:
         print_normal('Generate search space done: \'{}\'.'.format(file_path))
+
+def experiment_head(args):
+    '''list maximal trials' id and results'''
+    nni_config = Config(get_config_filename(args))
+    rest_port = nni_config.get_config('restServerPort')
+    rest_pid = nni_config.get_config('restServerPid')
+    if not detect_process(rest_pid):
+        print_error('Experiment is not running...')
+        return
+    running, response = check_rest_server_quick(rest_port)
+    if running:
+        response = rest_get(metric_data_latest_url(rest_port), 20)
+        if response is not None and check_response(response):
+            content = json.loads(response.text)
+            if int(args.num) > len(content):
+                print_error('Required number is too large.')
+                return
+            content = sorted(content, key=lambda x: float(x['data'][2: -2]), reverse=True)
+            list_trial = ''
+            for idx in range(int(args.num)):
+                list_trial += EXPERIMENT_RESULT_DETAIL_FORMAT.format(content[idx]['trialJobId'],
+                                                                     content[idx]['data'][1: -2])
+            print_normal(EXPERIMENT_RESULT_FORMAT.format('Maximal', list_trial))
+        else:
+            print_error('Get latest data failed...')
+    else:
+        print_error('Restful server is not Running')
+
+def experiment_tail(args):
```

Comment:
experiment_tail and experiment_head can be implemented with one function,  use the sort order as parameter.